addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18404,7 +18404,31 @@ msgctxt "#36050"
 msgid "On start"
 msgstr ""
 
-#empty strings from id 36051 to 36098
+#empty strings from id 36051 to 36094
+
+#. Label of setting "System / Display / Use 10 bit for SDR"
+#: system/settings/settings.xml
+msgctxt "#36095"
+msgid "Use 10 bit for SDR"
+msgstr ""
+
+#. Value for setting with label #36095 "System / Display / Use 10 bit for SDR"
+#: system/settings/settings.xml
+msgctxt "#36096"
+msgid "Auto detect"
+msgstr ""
+
+#. Value for setting with label #36095 "System / Display / Use 10 bit for SDR"
+#: system/settings/settings.xml
+msgctxt "#36097"
+msgid "Never"
+msgstr ""
+
+#. Value for setting with label #36095 "System / Display / Use 10 bit for SDR"
+#: system/settings/settings.xml
+msgctxt "#36098"
+msgid "Always"
+msgstr ""
 
```

Comment:
Similar again. **Always** already exists as String 20422 https://github.com/xbmc/xbmc/blob/7a043e7b6e777423f55a029c9d0f6b9c0dca33a3/addons/resource.language.en_gb/resources/strings.po#L13430

so this becomes

```
#. Value for setting with label #170 "Player / Videos / Adjust display refresh rate"
#. Value for setting with label #36095 "System / Display / Use 10 bit for SDR"
#: system/settings/settings.xml
msgctxt "#20422"
msgid "Always"
msgstr ""
```